pyfixest/report/visualize.py

```diff
@@ -235,7 +258,17 @@ def coefplot(
     )
 
 
-def _coefplot(
+def _coefplot(plot_backend, **plot_kwargs):
+    """Coefplot function that dispatches to the correct plotting backend."""
+    if plot_backend == "lets_plot":
+        return _coefplot_lets_plot(**plot_kwargs)
+    elif plot_backend == "matplotlib":
+        return _coefplot_matplotlib(**plot_kwargs)
+    else:
+        raise ValueError("plot_backend must be either 'lets_plot' or 'matplotlib'.")
```

Comment:
Here we could add a very simple test that triggers this error to `test_errors.py`. I can do this tomorrow morning.

Reply:
Yes! I was planning on adding it after the initial feedback 😄